<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scontrino</class>
 <widget class="QWidget" name="Scontrino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scontrino</string>
  </property>
  <widget class="QLabel" name="l_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data acquisto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_importo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Importo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_punti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Punti:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fumetti acquistati:</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_fumetti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>491</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conferma">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma e chiudi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tessera">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>480</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Aumenta punti su tessera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
